fossunited/fossunited/doctype/foss_event_cfp_submission/templates/foss_event_cfp_submission.html

```diff
@@ -0,0 +1,447 @@
+{% extends "templates/foss_base.html" %}
+
+{% block title %}{{ doc.talk_title }}{% endblock %}
+
+{% block page_content %}
+
+{% set cfp = frappe.db.get_value("FOSS Event CFP", linked_cfp, ["chapter", "anonymous_proposals", "allow_cfp_edit"], as_dict=True) %}
+
+<div class="container" id="head-cfp">
+    <div class="my-6">
+        {{ BreadCrumb(doc.route) }}
+    </div>
+
+    <div class="date-time-category-flex">
+        <div class="icon-text-container">
+            <img src="/assets/fossunited/icons/hash-02.svg">
+            {{ doc.category }}
+        </div>
+        <div class="icon-text-container">
+            <img src="/assets/fossunited/icons/hourglass-02.svg">
+            {{ doc.talk_duration }}
+        </div>
+        {% if doc.first_talk == "Yes" %}
```

Comment:
Should be a Boolean? 

```suggestion
        {% if doc.is_first_talk %}
```

Again, readability. Look for Uncle bob btw @harshtandiya 😀